### ISSUE TITLE
lr scheduler and gradient norm in FusedAdam

### ISSUE THREAD

**Issue by pkuyym** [NVIDIA/apex#608]
Does FusedAdam support lr scheduler and gradient norm? I can't find any document. Thanks!

**Comment by mcarilli** [comment]
`apex.optimizers.FusedAdam` should work with `torch.nn.utils.clip_grad_norm_` if that's what you're wondering.  It should also support LR scheduling although you may get a one-time warning about a step method being patched.

**Comment by pkuyym** [comment]
@mcarilli thanks.
